packages/jest-haste-map/src/index.js

```diff
@@ -227,11 +227,11 @@ class HasteMap extends EventEmitter {
     this._cachePath = HasteMap.getCacheFilePath(
       this._options.cacheDirectory,
       `haste-map-${this._options.name}`,
-      VERSION,
-      this._options.roots.join(':'),
-      this._options.extensions.join(':'),
-      this._options.platforms.join(':'),
-      options.mocksPattern || '',
+      VERSION +
```

Comment:
The version changes when we update Jest and wipes away the cache, so it isn't really breaking ;)